name.abuchen.portfolio.tests/src/name/abuchen/portfolio/model/SecurityTest.java

```diff
@@ -199,22 +198,22 @@ public void testExternalIdentifier()
         Security security = new Security();
 
         security.setName("Apple ORD");
-        assertEquals(security.getExternalIdentifier(), "Apple ORD");
+        assertThat(security.getExternalIdentifier(), is("Apple ORD"));
```

Comment:
I wonder: isn't that still the wrong order (if it is "expected, actual")?

Reply:
Okay, got me confused... 

It is "assertEquals(expected, actual)" and "assertThat(actual, matcher)"